### ISSUE TITLE
Switch -c results in TypeError

### ISSUE THREAD

**Issue by bitsgalore** [titusz/epubcheck#11]
I just tried to generate CSV output for a directory with some EPUBs. I used the following command:

```
epubcheck -c test.csv ./samples-test/
```

But this results in an TypeErrorr (output file is created, but it is empty):

```
TypeError: "delimiter" must be string, not bytes
```

Environment:

- Python 3.8.10
- Linux Mint 20.1 Ulyssa

**Comment by titusz** [comment]
I could not reproduce this issue, but it might be fixed by the latest release. Give it a try...

**Comment by bitsgalore** [comment]
Yep, it works now, again thanks for the fix (also good to see the upgrade to the 4.2.6 JAR)!
